src/components/Timeline/src/StepLabel.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import StepIcon, { StepIconProps } from './StepIcon';
+import { StepComponentProps } from './StepComponentProps';
+
+export interface StepLabelProps extends StepComponentProps {
+  /**
+   * The label displayed in the step icon.
+   */
+  icon?: number;
```

Comment:
Why is this property of type `number`? Feel like this should be at least `string | number`...

Reply:
I've updated the type.